test/functional/feature_rbf.py

```diff
@@ -714,5 +717,33 @@ def test_replacement_relay_fee(self):
         tx.vout[0].nValue -= 1
         assert_raises_rpc_error(-26, "insufficient fee", self.nodes[0].sendrawtransaction, tx.serialize().hex())
 
+    def test_fullrbf(self):
+        txid = self.wallet.send_self_transfer(from_node=self.nodes[0])['txid']
+        self.generate(self.nodes[0], 1)
+        confirmed_utxo = self.wallet.get_utxo(txid=txid)
```

Comment:
Sorry for the merge conflict, but now you might be able to write this a bit shorter:

```suggestion
        confirmed_utxo = self.make_utxo(...)
```

Let me know if you have any questions.

Reply:
Oh nvm. There is no merge conflict. I can fix this up after merge, if needed.